chrome/settings-navigation.json

```diff
@@ -82,7 +82,8 @@
                 {
                     "method": "isOrgAdmin",
```

Comment:
There is still the org admin reference. Shouldn't this dondition be removed entirely?

Reply:
I think that this should be `isEntitled`

Reply:
Updated. Can you take another look. I want to make sure that this checks insights OR cost, not AND